src/NumericalAlgorithms/Interpolation/InitializeInterpolationTarget.hpp

```diff
@@ -3,21 +3,87 @@
 
 #pragma once
 
-#include <algorithm>
+#include <cstddef>
+#include <tuple>
+#include <unordered_set>
 
 #include "DataStructures/DataBox/DataBox.hpp"
 #include "DataStructures/DataBox/DataBoxTag.hpp"
-#include "DataStructures/Variables.hpp"
-#include "Domain/Tags.hpp"
-#include "NumericalAlgorithms/Interpolation/Tags.hpp"
-#include "Parallel/ConstGlobalCache.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/Tags.hpp"                             // IWYU pragma: keep
+#include "NumericalAlgorithms/Interpolation/Tags.hpp"  // IWYU pragma: keep
+#include "Utilities/Requires.hpp"
+#include "Utilities/TMPL.hpp"
 #include "Utilities/TaggedTuple.hpp"
+#include "Utilities/TypeTraits.hpp"
+
+/// \cond
+// IWYU pragma: no_forward_declare db::DataBox
+namespace Parallel {
+template <typename Metavariables>
+class ConstGlobalCache;
+}  // namespace Parallel
+namespace Tags {
+template <typename TagsList>
+struct Variables;
+}  // namespace Tags
+/// \endcond
 
 namespace intrp {
 
 /// Holds Actions for Interpolator and InterpolationTarget.
 namespace Actions {
 
+// The purpose of the functions and metafunctions in this
+// namespace is to allow InterpolationTarget::compute_target_points
+// to omit an initialize function and a initialization_tags
+// type alias if it doesn't add anything to the DataBox.
+namespace initialize_interpolation_target_detail {
+
+// Sets value to initialization_tags, or
+// to empty list if initialization_tags is not defined.
+template <typename T, typename = cpp17::void_t<>>
+struct initialization_tags {
+  using value = tmpl::list<>;
```

Comment:
I think `value` should be `type` since it's a type and not a value. The STL uses `type` for what you've called "value" and `value` for variables.